### ISSUE TITLE
HELP! GPU memory overflow during testing.

### ISSUE THREAD

**Issue by shenxiaowrj** [CVPR2023-3D-Occupancy-Prediction/CVPR2023-3D-Occupancy-Prediction#6]
My partner and I both encountered GPU memory overflow during testing.
The error message is as follows:
![d02dfafc1ac158be4215c9b158fe66b](https://user-images.githubusercontent.com/100119921/224598689-95926682-f5e7-4b9e-9477-18c233ef2ecf.jpg)

Because the error message is not saved, the image is sent.

**Comment by YWonchall** [comment]
Hi, What is the size of your GPU memory? Is it enough to train the baseline model?

**Comment by shenxiaowrj** [comment]
> 

Hi! I used the [bevformer_small_occ.py](https://github.com/CVPR2023-3D-Occupancy-Prediction/CVPR2023-3D-Occupancy-Prediction/blob/main/projects/configs/bevformer/bevformer_small_occ.py) to train the model on my RTX 3090 (24G memory).
During the training process, the model is trained steadily. But when I was testing, when I got to the 3504th scene, the memory was beyond what my machine could handle.
But interestingly, my friend used two RTX 3090s to test the model, and his program was stopped when processing the 5900th scene because of memory issues.
